examples/multizone-rule/variables.tf

```diff
@@ -27,3 +27,19 @@ variable "resource_tags" {
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
+##############################################################
+# CBR
+##############################################################
+
+variable "rule_description" {
+  type        = string
+  description = "(Optional, String) The description of the rule"
+  default     = null
+}
+
+variable "enforcement_mode" {
+  type        = string
+  description = "(String) The rule enforcement mode"
+  default     = "enabled"
```

Comment:
this value changed to `report` in a recent PR internally - please check that Pr to ensure all of those changes made it into this PR